trading_calendars/exchange_calendar_xnse.py

```diff
@@ -0,0 +1,366 @@
+    '2010-11-05',
+    '2010-11-17',
+    '2010-12-17',
+    '2011-01-26',
+    '2011-03-02',
+    '2011-04-12',
+    '2011-04-14',
+    '2011-04-22',
+    '2011-08-15',
+    '2011-08-31',
+    '2011-09-01',
+    '2011-10-06',
+    '2011-10-26',
+    '2011-10-27',
+    '2011-11-07',
+    '2011-11-10',
+    '2011-12-06',
+    '2012-01-26',
+    '2012-02-20',
+    '2012-03-08',
+    '2012-04-05',
+    '2012-04-06',
+    '2012-05-01',
+    '2012-08-15',
+    '2012-08-20',
+    '2012-09-19',
+    '2012-10-02',
+    '2012-10-24',
+    '2012-10-26',
+    '2012-11-13',
+    '2012-11-14',
+    '2012-11-28',
+    '2012-12-25',
+    '2013-03-27',
+    '2013-03-29',
+    '2013-04-19',
+    '2013-04-24',
+    '2013-05-01',
+    '2013-08-09',
+    '2013-08-15',
+    '2013-09-09',
+    '2013-10-02',
+    '2013-10-16',
+    '2013-11-04',
+    '2013-11-15',
+    '2013-12-25',
+    '2014-02-27',
+    '2014-03-17',
+    '2014-04-08',
+    '2014-04-14',
+    '2014-04-18',
+    '2014-04-24',
+    '2014-05-01',
+    '2014-07-29',
+    '2014-08-15',
+    '2014-08-29',
+    '2014-10-02',
+    '2014-10-03',
+    '2014-10-06',
+    '2014-10-15',
+    '2014-10-23',
+    '2014-10-24',
+    '2014-11-04',
+    '2014-11-06',
+    '2014-12-25',
+    '2015-01-26',
+    '2015-02-17',
+    '2015-03-06',
+    '2015-04-02',
+    '2015-04-03',
+    '2015-04-14',
+    '2015-05-01',
+    '2015-09-17',
+    '2015-09-25',
+    '2015-10-02',
+    '2015-10-22',
+    '2015-11-11',
+    '2015-11-12',
+    '2015-11-25',
+    '2015-12-25',
+    '2016-01-26',
+    '2016-03-07',
+    '2016-03-24',
+    '2016-03-25',
+    '2016-04-14',
+    '2016-04-15',
+    '2016-04-19',
+    '2016-07-06',
+    '2016-08-15',
+    '2016-09-05',
+    '2016-09-13',
+    '2016-10-11',
+    '2016-10-12',
+    '2016-10-31',
+    '2016-11-14',
+    '2017-01-26',
+    '2017-02-24',
+    '2017-03-13',
+    '2017-04-04',
+    '2017-04-14',
+    '2017-05-01',
+    '2017-06-26',
+    '2017-08-15',
+    '2017-08-25',
+    '2017-10-02',
+    '2017-10-20',
+    '2017-12-25',
+    '2018-01-26',
+    '2018-02-13',
+    '2018-03-02',
+    '2018-03-29',
+    '2018-03-30',
+    '2018-05-01',
+    '2018-08-15',
+    '2018-08-22',
+    '2018-09-13',
+    '2018-09-20',
+    '2018-10-02',
+    '2018-10-18',
+    '2018-11-07',
+    '2018-11-08',
+    '2018-11-23',
+    '2018-12-25',
+    '2019-01-26',
+    '2019-02-13',
+    '2019-02-14',
+    '2019-03-02',
+    '2019-03-04',
+    '2019-03-21',
+    '2019-03-29',
+    '2019-04-04',
+    '2019-04-13',
+    '2019-04-14',
+    '2019-04-17',
+    '2019-04-19',
+    '2019-04-29',
+    '2019-05-01',
+    '2019-06-05',
+    '2019-08-12',
+    '2019-08-15',
+    '2019-09-02',
+    '2019-09-10',
+    '2019-10-02',
+    '2019-10-08',
+    '2019-10-27',
+    '2019-10-28',
+    '2019-11-12',
+    '2019-12-25',
+    '2020-02-21',
+    '2020-03-09',
+    '2020-04-02',
+    '2020-04-06',
+    '2020-04-10',
+    '2020-04-14',
+    '2020-04-1',
+    '2020-05-01',
+    '2020-07-31',
+    '2020-08-20',
```

Comment:
I don't see this one on the NSE website's list, what was the reason for the closure?